src/api/common/schemas/balance-sheet-options.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "balance-sheet-options",
+  "description": "Options for getBalanceSheet",
+  "type": "object",
+  "properties": {
+    "hotwallet": {
```

Comment:
That's ok, `['address']` is fine if it allows the name and syntax to indicate the possibility of adding more addresses.